src/Employer/Employer.Web/Orchestrators/VacancyManageOrchestrator.cs

```diff
@@ -6,21 +6,28 @@
 using Esfa.Recruit.Employer.Web.ViewModels;
 using Esfa.Recruit.Vacancies.Client.Domain.Entities;
 using Esfa.Recruit.Vacancies.Client.Domain.Exceptions;
-using Esfa.Recruit.Vacancies.Client.Infrastructure.Client;
+using Esfa.Recruit.Vacancies.Client.Domain.Services;
 
 namespace Esfa.Recruit.Employer.Web.Orchestrators
 {
     public class VacancyManageOrchestrator
     {
         private readonly DisplayVacancyViewModelMapper _vacancyDisplayMapper;
+        private readonly ITimeProvider _timeProvider;
 
-        public VacancyManageOrchestrator(DisplayVacancyViewModelMapper vacancyDisplayMapper)
+        public VacancyManageOrchestrator(DisplayVacancyViewModelMapper vacancyDisplayMapper, ITimeProvider timeProvider)
         {
             _vacancyDisplayMapper = vacancyDisplayMapper;
+            _timeProvider = timeProvider;
         }
 
         public async Task<ManageVacancy> GetVacancyDisplayViewModelAsync(Vacancy vacancy)
         {
+            if (vacancy.ClosingDate.HasValue && vacancy.ClosingDate.Value <= _timeProvider.Now)
```

Comment:
We must remember to delete this bit once the job has been created to close vacancies. Worth a comment?

Reply:
Yep, I have kept a note. We can maybe put it into the other close vacancy ticket as well.